src/main/native/jgskit_resource.rc

```diff
@@ -27,13 +27,13 @@ BEGIN
         BEGIN
             VALUE "CompanyName", "IBM\0"
             VALUE "FileDescription", "Native JGSKIT runtime library\0"
-            VALUE "FileVersion", "21\0"
+            VALUE "FileVersion", "23\0"
 	        VALUE "Build Increment", "0\0"
             VALUE "InternalName", "jgskit.dll\0"
-            VALUE "LegalCopyright", "(C) Copyright IBM Corp. 2023. Licensed under the Apache License 2.0.\0"
+            VALUE "LegalCopyright", "(C) Copyright IBM Corp. 2023, 2024. Licensed under the Apache License 2.0.\0"
```

Comment:
I think we should update the copyright at the top of this file too.

Reply:
Yes agreed, updated the copyright at top of file with most recent commit 4dc696a